examples/text-replacer/content.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+// Replace text on the page using a static list of patterns
+function textReplacer(replacements) {
```

Comment:
I think having this method is a bit confusing. I'd just move its two lines into the callback for the fetcher, and expand the comment:

```js
// Get the patterns to replace from storage, then build a regex from them and replace all text on the page.
```